examples/README.md

```diff
@@ -27,3 +27,30 @@ node dist/example.js
 ```
 
 and it will show the usage of the script with all the actions that you can do.
+
+## Example commands
+
+```
+n dist/example.js create-wallet 1
```

Comment:
is this `n` supposed to be `node`? I’m not familar with the `n` and think we should use `node`